plugin/pkg/scheduler/factory/factory.go

```diff
@@ -147,6 +157,84 @@ func NewConfigFactory(client *client.Client, schedulerName string, hardPodAffini
 		},
 	)
 
+	c.PVLister.Store, c.pvPopulator = cache.NewInformer(
+		c.createPersistentVolumeLW(),
+		&api.PersistentVolume{},
+		0,
+		cache.ResourceEventHandlerFuncs{
+			AddFunc: func(_ interface{}) {
+				c.EquivalencePodCache.SendClearAllCacheReq()
+			},
+			UpdateFunc: func(_, _ interface{}) {
+				c.EquivalencePodCache.SendClearAllCacheReq()
+			},
+			DeleteFunc: func(_ interface{}) {
+				c.EquivalencePodCache.SendClearAllCacheReq()
+			},
```

Comment:
Maybe we can some up with an API, to only clear the effected predicate from the cache in such cases.